Sources/Composable.swift

```diff
@@ -208,8 +248,48 @@ extension Special: Composable {
     ///
     /// - parameter to:        The attributed string to which to append the receiver.
     /// - parameter baseStyle: The style to use.
-    public func append(to attributedString: NSMutableAttributedString, baseStyle: StringStyle) {
+    public func append(to attributedString: NSMutableAttributedString, baseStyle: StringStyle, isLastElement: Bool) {
         description.append(to: attributedString, baseStyle: baseStyle)
```

Comment:
Should `isLastElement` be passed along here?

Reply:
I looked at this. The method is part of the `Composable` protocol, but the `Special` type doesn't care about it, so it ignores it.